charts/renku/values.yaml

```diff
@@ -589,22 +593,18 @@ notebooks:
     #     enabled: true
 
 tests:
+  enabled: false
   image:
-    repository: renku/tests
-    tag: 'latest'
-  ## User accounts for running `helm test`
-  ## Replace passwords with random strings in a production setup
-  #users_json:
-  #  - username: bwayne
-  #    firstName: Bruce
-  #    lastName: Wayne
-  #    email: bwayne@example.com
-  #    password: IamBatman
-  #  - username: ckent
-  #    firstName: Clark
-  #    lastName: Kent
-  #    email: ckent@example.com
-  #    password: IamSuperman
+    repository: degano/tests
+    tag: '0.7.4-8497790'
+  ## User account for running `helm test`
+  #parameters:
```

Comment:
sorry to ask for one more thing but could you add the rest of the parameters here to serve as documentation?

Reply:
Sure!